### DISCUSSION TITLE
Global motion sensor

### DISCUSSION THREAD

**Discussion by ronaldt80** [fuatakgun/eufy_security#225] [General]
Out of curiousity, what is the purpose of the global motion sensor? When I first saw it I expected it to combine both the person and non-person motion sensor but it seems to mirror the non-person sensor.

My 2C cam recognizes/reports person motion during day time and all motion at night - resulting in 2 different sensors triggering.

**Comment by fuatakgun** [comment]
In latest version of integration, it will be on whenever motion or person or pet is on.

I have 2c cameras and whenever any of motion sensors (motion and person) are on, global version is also turning into on.

**Reply by ronaldt80** [reply]
Ah ok, so it intends to do what I expected. Will check tomorrow, just updated and described behaviour is based on the previous version. Thanks!
